group/models.py

```diff
@@ -142,10 +142,18 @@ class MeetingMinutes(AbstractReport):
     """
     Meeting minutes content type.
     """
-    panels = AbstractReport.panels
+    date = models.DateField(blank=False)
+    summary = models.TextField(null=False, blank=False)
```

Comment:
I think these two lines can be removed. Since this class extends `AbstractReport` it should already have these fields and you should still be able to manually set the panels without them, the same as you're doing. The joys of inheritance!

Reply:
Removed this, but added code to make linking to a page required. I didn't realize that the summary linked to nothing, even if no page was chosen!